### ISSUE TITLE
Configuring data-purger with Cassandra causes "no viable alternative at input" error

### ISSUE THREAD

**Issue by Andrewpk** [cortexproject/cortex#2718]
Key Error:
`level=error ts=2020-06-11T04:32:03.920695643Z caller=cortex.go:348 msg="module failed" module=data-purger error="invalid service state: Failed, expected: Running, failure: line 1:56 no viable alternative at input 'value' (...FROM delete_requests WHERE hash = [?] value...)"`

Logs:
```
level=warn ts=2020-06-11T04:32:03.680094876Z caller=experimental.go:19 msg="experimental feature in use" feature="Cassandra Backend"
level=info ts=2020-06-11T04:32:03.85185591Z caller=server.go:179 http=[::]:80 grpc=[::]:9095 msg="server listening on addresses"
level=debug ts=2020-06-11T04:32:03.852008304Z caller=api.go:93 msg="api: registering route" methods= path=/config auth=false
level=debug ts=2020-06-11T04:32:03.852038883Z caller=api.go:93 msg="api: registering route" methods= path=/ auth=false
level=warn ts=2020-06-11T04:32:03.852239642Z caller=experimental.go:19 msg="experimental feature in use" feature="Cassandra Backend"
level=warn ts=2020-06-11T04:32:03.881851928Z caller=experimental.go:19 msg="experimental feature in use" feature="Cassandra Backend"
level=warn ts=2020-06-11T04:32:03.91847156Z caller=experimental.go:19 msg="experimental feature in use" feature="Delete series API"
level=debug ts=2020-06-11T04:32:03.918555767Z caller=api.go:93 msg="api: registering route" methods=PUT,POST path=/prometheus/api/v1/admin/tsdb/delete_series auth=true
level=debug ts=2020-06-11T04:32:03.91862478Z caller=api.go:93 msg="api: registering route" methods=GET path=/prometheus/api/v1/admin/tsdb/delete_series auth=true
level=debug ts=2020-06-11T04:32:03.918661007Z caller=api.go:93 msg="api: registering route" methods=PUT,POST path=/api/prom/api/v1/admin/tsdb/delete_series auth=true
level=debug ts=2020-06-11T04:32:03.918694792Z caller=api.go:93 msg="api: registering route" methods=GET path=/api/prom/api/v1/admin/tsdb/delete_series auth=true
level=debug ts=2020-06-11T04:32:03.918728483Z caller=api.go:93 msg="api: registering route" methods= path=/services auth=false
level=info ts=2020-06-11T04:32:03.918749868Z caller=main.go:119 msg="Starting Cortex" version="(version=1.1.0, branch=HEAD, revision=9db28c16)"
level=debug ts=2020-06-11T04:32:03.918834671Z caller=module_service.go:48 msg="module waiting for initialization" module=data-purger waiting_for=store
level=info ts=2020-06-11T04:32:03.918861843Z caller=module_service.go:58 msg=initialising module=runtime-config
level=info ts=2020-06-11T04:32:03.918866967Z caller=module_service.go:58 msg=initialising module=store
level=info ts=2020-06-11T04:32:03.918900743Z caller=module_service.go:58 msg=initialising module=data-purger
level=info ts=2020-06-11T04:32:03.918894943Z caller=manager.go:109 msg="runtime config disabled: file not specified"
level=debug ts=2020-06-11T04:32:03.920691853Z caller=module_service.go:80 msg=stopping module=runtime-config
level=error ts=2020-06-11T04:32:03.920695643Z caller=cortex.go:348 msg="module failed" module=data-purger error="invalid service state: Failed, expected: Running, failure: line 1:56 no viable alternative at input 'value' (...FROM delete_requests WHERE hash = [?] value...)"
level=info ts=2020-06-11T04:32:03.920776753Z caller=module_service.go:90 msg="module stopped" module=runtime-config
level=debug ts=2020-06-11T04:32:03.920796155Z caller=module_service.go:80 msg=stopping module=store
level=info ts=2020-06-11T04:32:03.920902333Z caller=module_service.go:90 msg="module stopped" module=store
level=info ts=2020-06-11T04:32:03.920998126Z caller=server_service.go:48 msg="server stopped"
level=info ts=2020-06-11T04:32:03.921043399Z caller=cortex.go:337 msg="Cortex stopped"
level=error ts=2020-06-11T04:32:03.921102319Z caller=log.go:140 msg="error running cortex" err="failed services\ngithub.com/cortexproject/cortex/pkg/cortex.(*Cortex).Run\n\t/go/src/github.com/cortexproject/cortex/pkg/cortex/cortex.go:388\nmain.main\n\t/go/src/github.com/cortexproject/cortex/cmd/cortex/main.go:121\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1357"
```

Configuration arguments:
```
        - -target=data-purger
        - -purger.enable=true
        - -purger.num-workers=2
        - -purger.object-store-type=filesystem
        - -deletes.store=cassandra
        - -schema-config-file=/etc/cortex/schema.yaml
        - -store.engine=chunks
        - -cassandra.addresses=cassandra address
        - -cassandra.keyspace=cortex
        - -cassandra.timeout=1000ms
        - -cassandra.replication-factor=1
```

**Comment by sandeepsukhani** [comment]
It seems the problem was with the building of query in Cassandra client which was fixed after 1.1.0 release [here](https://github.com/cortexproject/cortex/commit/c3b0724f24264c24e188e61fc8c312703d0daf91).
Also please use the latest master build or any build which contains [this](https://github.com/cortexproject/cortex/commit/85f93ff4abd7af81a5eeeae0e07379dd6ff2071d) commit which does the table provisioning for storing delete requests.

**Comment by pracucci** [comment]
Closing because should have been fixed by https://github.com/cortexproject/cortex/commit/c3b0724f24264c24e188e61fc8c312703d0daf91.
